javascript-objects/main.js

```diff
@@ -0,0 +1,30 @@
+const student = {
+  firstName: 'Kitlong',
+  lastName: 'Lai',
+  age: 26
+};
+const fullName = student.firstName + student.lastName;
+console.log('fullName:' + fullName);
+student.livesInIrvine = true;
+student.previousOccupation = 'Administrator';
+console.log('livesInIrvine:' + student.livesInIrvine);
+console.log('previousOccupation:' + student.previousOccupation);
+
+const vehicle = {
+  make: 'Porsche',
+  model: '992 GT3RS',
+  year: 2023
+};
+vehicle.color = 'chalk';
```

Comment:
Let's make sure to practice using bracket notation (lines 18-21) instead of dot notation here.